<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageView</class>
 <widget class="QWidget" name="ImageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>526</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>ImageView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QComboBox" name="deviceListComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>48</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Device List</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: rgb(200,200,200);
    background-color: rgba(0,0,0,80);
}

QComboBox:!editable {
    color: rgb(200,200,200);
    background-color: rgba(0,0,0,80);
}

QComboBox:hover {
    background-color: rgba(0,0,0,120);
    border: 2px solid rgba(120,0,0,180);
}

QComboBox::drop-down {
    background-color: rgba(0,0,0,120);
    border: none;
	image: url(:/ICON/Resources/bullet_arrow_left.ico);
}

QComboBox::drop-down:on {
	image: url(:/ICON/Resources/bullet_arrow_down.ico);
}

QComboBox ListView {
    border: none;
	background-color: qlineargradient(spread:pad, x1:0.903, y1:0.199, x2:0, y2:0.999, stop:0 rgba(0, 0, 0, 255), stop:0.875 rgba(122, 122, 122, 255), stop:1 rgba(255, 255, 255, 255));
}

QComboBox QAbstractItemView {
    border: none;
    selection-color: lightblue;
    selection-background-color: rgba(0,0,0,200);
}</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="HoverMenuButton" name="settingButton" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#settingButton {
	background-image: url(:/ICON/Resources/settings.ico);
}

#settingButton:hover {
	padding-left: 2px;
    padding-top: 2px;
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>198</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="ControlPanel" name="controlPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>64</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>controlpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HoverMenuButton</class>
   <extends>QWidget</extends>
   <header>hovermenubutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="videorecorder.qrc"/>
 </resources>
 <connections/>
</ui>
